client/my-sites/plans/trials/business-trial-plans/index.tsx

```diff
@@ -23,11 +28,15 @@ export function BusinessTrialPlans( props: BusinessTrialPlansProps ) {
 
 			const planPath = getPlanPath( upgradePlanSlug ) ?? '';
 
-			const checkoutUrl = isBusinessPlan( upgradePlanSlug )
-				? getTrialCheckoutUrl( { productSlug: planPath, siteSlug } )
+			const cartItemForStorageAddOn = cartItems?.find(
+				( items ) => items.product_slug === PRODUCT_1GB_SPACE
+			);
+
+			const testUrl = isBusinessPlan( upgradePlanSlug )
```

Comment:
Just a quick drive-by comment: is there a specific reason to name this `testUrl`? :D

Reply:
🤦‍♂️  Totally my mistake! It was meant for logging and I completely spaced on switching it back to checkout Url. Thanks for catching this -- I'll do a quick sweep of my code for any other oddities 😅

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/83771/commits/5012539e57161726a6618c3f3f030ef0fc9e283a